### PR TITLE
new examples

### PR DESCRIPTION
**Pull request by mvartakAtVerta**
E2E examples with new convenience functions for registry ingest

### CODE REVIEW

File: client/workflows/demos/registry/create-standard-model.ipynb [hunk 1]
```diff
@@ -0,0 +1,135 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Creating a Standard Model on Verta\n",
+    "\n",
+    "Within Verta, a \"Model\" can be any arbitrary function: a traditional ML model (e.g., sklearn, PyTorch, TF, etc); a function (e.g., squaring a number, making a DB function etc.); or a mixture of the above (e.g., pre-processing code, a DB call, and then a model application.) See more [here](https://docs.verta.ai/verta/collaboration/concepts).\n",
```
Comment (convoliution) [inline client/workflows/demos/registry/create-standard-model.ipynb]: The link at the end of this line
```markdown
See more [here](https://docs.verta.ai/verta/collaboration/concepts)
```
appears to be incorrect!

File: client/workflows/demos/registry/multiple-framework-example-end-to-end.ipynb [hunk 2]
```diff
@@ -0,0 +1,499 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Deploying Models with Multiple ML Frameworks on Verta\n",
+    "\n",
+    "Within Verta, a \"Model\" can be any arbitrary function: a traditional ML model (e.g., sklearn, PyTorch, TF, etc); a function (e.g., squaring a number, making a DB function etc.); or a mixture of the above (e.g., pre-processing code, a DB call, and then a model application.) See more [here](https://docs.verta.ai/verta/collaboration/concepts).\n",
```
Comment (convoliution) [inline client/workflows/demos/registry/multiple-framework-example-end-to-end.ipynb]: The link at the end of this line
```markdown
See more [here](https://docs.verta.ai/verta/collaboration/concepts)
```
appears to be incorrect!

File: client/workflows/demos/registry/sklearn-census-end-to-end.ipynb [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_version_v2 = registered_model.create_standard_model(\n",
+    "    model_cls=CensusIncomeClassifier,\n",
+    "    environment=Python(requirements=[\"scikit-learn\"]),\n",
+    "    artifacts={\"model.pkl\" : \"moodel.pkl\"},\n",
```
Comment (convoliution) [inline client/workflows/demos/registry/sklearn-census-end-to-end.ipynb]: I think we're missing a
```python
with open("model.pkl", "rb") as f:
    cloudpickle.dump(model, f)
```
somewhere. Or, this line could be
```suggestion
    "    artifacts={\"model.pkl\" : model},\n",
```

Also, "moodel.pkl" might be a typo 🐄

File: client/workflows/demos/registry/sklearn-census-end-to-end.ipynb [hunk 4]
```diff
@@ -0,0 +1,383 @@
...
+    "from verta.registry import VertaModelBase\n",
+    "\n",
+    "class CensusIncomeClassifier(VertaModelBase):\n",
+    "    def __init__(self, artifacts):\n",
+    "        self.model = cloudpickle.load(artifacts[\"serialized_model\"])\n",
+    "        \n",
+    "    def predict(self, batch_input):\n",
+    "        results = []\n",
+    "        for one_input in batch_input:\n",
+    "            sself.model.predict(one_input)\n",
```
Comment (convoliution) [inline client/workflows/demos/registry/sklearn-census-end-to-end.ipynb]:
```suggestion
    "            self.model.predict(one_input)\n",
```

**Review by convoliution — CHANGES_REQUESTED** [review]
Most of the notebooks (not just the once I commented on) have a probably-erroneous link to `collaboration/concepts`

I've compiled more suggestions here (b/c editing locally was easier than reading the notebooks in GitHub 😅):
https://github.com/VertaAI/modeldb/pull/2481

File: client/workflows/demos/registry/create-standard-model.ipynb (hunk 1 above)
Comment (mvartakAtVerta) [reply client/workflows/demos/registry/create-standard-model.ipynb]: Gitbook copy fun. fixed

File: client/workflows/demos/registry/multiple-framework-example-end-to-end.ipynb (hunk 2 above)
Comment (mvartakAtVerta) [reply client/workflows/demos/registry/multiple-framework-example-end-to-end.ipynb]: Fixed

**Review by convoliution — CHANGES_REQUESTED** [review]
Last remaining change I'd push for is uncommenting `import os` in the
```python
os.environ[...] = 
```
cells, because `os` is needed for the
```python
client = Client(os.environ['VERTA_HOST'])
```

That, and a couple other small formatting suggestions, in https://github.com/VertaAI/modeldb/pull/2481

**Review by convoliution — APPROVED** [review]

**Pull request merged**